src/core/utils.ts

```diff
@@ -113,7 +113,13 @@ export function reverseDeepMerge(dest, source = undefined) {
         if (isFunction(src[srcArg])) {
           dest[srcArg] = src[srcArg];
         } else {
-          dest[srcArg] = clone(src[srcArg]);
+          try {
```

Comment:
Also, I considered leaving a console.log inside the catch but it seemed too verbose.